zerver/forms.py

```diff
@@ -217,7 +216,7 @@ def save(self,
 
         if user is not None:
             token = token_generator.make_token(user)
-            uid = urlsafe_base64_encode(force_bytes(user.id))
+            uid = urlsafe_base64_encode(str(user.id).encode('utf8'))
```

Comment:
I'm a bit confused about this line -- what type is `user.id`? I'd have thought it was an int. But then why did the old code work at all -- why didn't it get to the `raise TypeError...` line?

Reply:
I checked that the type of `user.id` is `int` here. But the `force_bytes` here is imported from `django.utils.encoding` that's why its working. Upon import the `force_bytes` from `zerver.lib.str_utils` we get the error.

Reply:
We can probably just leave alone any code that uses the Django force_bytes, or if we do want to fix these, it's important to have good tests.